scripts/extract_images.sh

```diff
@@ -1,16 +1,16 @@
 #!/bin/bash
 
 CHART_DIR=$1
-CHARTS_DIRS=`find $CHART_DIR -type d -exec test -e '{}'/values.yaml \; -print`
+CHARTS_DIRS=$(find "$CHART_DIR" -type d -exec test -e '{}'/values.yaml \; -print)
 TMP_IMAGE_FILE=/tmp/imageslist.txt
 
 if [ -e imagelist.txt ]; then
 	rm imagelist.txt
 fi
 for chart in $CHARTS_DIRS; do
-	helm template --values $chart/values.yaml $chart/ | yq -r "..|.image?" | sort -u | grep -v "null"  > $TMP_IMAGE_FILE
+	helm template --values "$chart"/values.yaml "$chart"/ | yq -r "..|.image?" | sort -u | grep -v "null"  > $TMP_IMAGE_FILE
 	sed --in-place '/---/d' $TMP_IMAGE_FILE
-	cat $TMP_IMAGE_FILE | sort -u > $chart/imagelist.txt
+	sort -u -o "$chart"/imagelist.txt "$chart"/imagelist.txt
 	# Unified imagelist.txt file. Containing image from all charts.
-	cat $chart/imagelist.txt >> imagelist.txt
+	cat "$chart"/imagelist.txt >> imagelist.txt
 done
```

Comment:
I'm just fixing the issue reported by the shellcheck. In a future PR I plan to add the  `set -euo pipefail` line and fix other possible issue due the change. I do not want to mix changes in the same PR.